.storybook-s2/docs/Migrating.jsx

```diff
@@ -0,0 +1,770 @@
+import { style } from '../../packages/@react-spectrum/s2/style/spectrum-theme' with {type: 'macro'};
+import {Link} from '@react-spectrum/s2';
+import {P, Code, Pre, H3, H2} from './typography';
+
+export function Migrating() {
+  return (
+    <div className={style({maxWidth: 'lg', marginX: 'auto', fontFamily: 'sans'})}>
+      <div className={style({marginX: 48})}>
+        <h1 className={style({fontSize: 'heading-2xl', color: 'heading', marginBottom: 48})}>
+          Migrating to Spectrum 2
+        </h1>
+        <P>An automated upgrade assistant is available by running the following command in the project you want to upgrade:</P>
+        <Pre>npx @react-spectrum/codemods s1-to-s2</Pre>
+
+        <P>For cases that the upgrade assistant doesn't handle automatically or where you'd rather upgrade some components manually, use the guide below.</P>
+
+        <P>Note that [PENDING] indicates that future changes will occur before the final release, and the current solution should be considered temporary.</P>
+
+        <H2>Components</H2>
+        <H3>All components</H3>
+        <ul className="sb-unstyled">
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Update imports to use the <Code>@react-spectrum/s2</Code> package instead of <Code>@adobe/react-spectrum</Code> or individual packages like <Code>@react-spectrum/button</Code></li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Update <Link href="https://react-spectrum.adobe.com/react-spectrum/styling.html#style-props">style props</Link> to use the <Link href="/?path=/docs/style-macro--docs">style macro</Link> instead. See the 'Style props' section below</li>
```

Comment:
Nit: Would be nice to have these sections be anchors for easy navigation/linking others to specific sections but that can be an enhancement for the future.

Reply:
Added this. Had to get a bit hacky because of the iframe.